source/b_usermanagement.rst

```diff
@@ -0,0 +1,68 @@
+.. _OrganisationsAnchor:
+
+=============
+Organisations
+=============
+
+All data in Lizard is linked to an organisation.
+Organisations are the backbone of our authorisation model.
+There are three types of authorisation options that can be applied to your data: 
+
+* **Public**: everyone can see your data 
+* **Common**: everyone with login credentials to Lizard can see your data 
+* **Private**: everyone with login credentials to Lizard AND user rights to your organisation can see your data
```

Comment:
"private" en "everyone" in dezelfde zin is wat raar :-)
=> "only"?

Reply:
Dit is een stukje logica... Je moet zowel in kunnen loggen ALS rechten hebben op die organisatie. Ik zou zo niet weten hoe we dit netter kunnen verwoorden.